release_util/management/commands/run_migrations.py

```diff
@@ -153,11 +31,12 @@ class Command(BaseCommand):
 
     success:
     - migration: [app1, 0001_initial]
-      duration: 3.45
-      output: All good!
+      output: Applying app1.0001_initial... OK
```

Comment:
Since all migrations are running together, there's no way of knowing how long individual migrations took (in the way this script is written). However, on review, I need a top-level duration of the `migrate` command recorded at the top-level of this data. I will fix that.